README.md

```diff
@@ -1,104 +1,216 @@
-# ntStat
+ntStat is a toolkit that uses Bloom filters to track both *k*-mer count and depth information for use in downstream applications. ntStat also models the *k*-mer count histogram using a mixture model, and infers valuable insights about the genome, sequencing data, and individual *k*-mers, *de novo*.
```

Comment:
Think it would make sense to give some examples about the insights here, just to make sure we pull in any prospective users from the start

Reply:
Addressed in 2057c6844555100d86789b492ba3d2a316aa628a